src/translate_c.zig

```diff
@@ -5768,6 +5793,11 @@ fn parseCPrimaryExpr(c: *Context, m: *MacroCtx, scope: *Scope) ParseError!Node {
             if (builtin_typedef_map.get(mangled_name)) |ty| return Tag.type.create(c.arena, ty);
             const identifier = try Tag.identifier.create(c.arena, mangled_name);
             scope.skipVariableDiscard(identifier.castTag(.identifier).?.data);
+            refs_var: {
+                const ident_node = c.global_scope.sym_table.get(slice) orelse break :refs_var;
+                const var_decl_node = ident_node.castTag(.var_decl) orelse break :refs_var;
+                m.refs_var_decl = !var_decl_node.data.is_const;
```

Comment:
Slight problem here:
```c
int a = 0;
const int b = 1;

#define foo a + b
```
```zig
pub export var a: c_int = 0;
pub export const b: c_int = 1;

pub const foo = a + b;
```

```suggestion
                if (!var_decl_node.data.is_const) m.refs_var_decl = true;
```

Reply:
Nice catch!